AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/QuerySortBy+SQLite.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+import SQLite
+
+extension QuerySortBy {
+    var sqlOrderBy: String {
+        switch self {
+        case .ascending(let key):
+            return "\(key.stringValue) ASC"
```

Comment:
1. it should be `key.columnName` I believe, otherwise you won't get the escaped name with the namespace. Refer to existing SQL generation for a reference.
2. the code is using lowercase for SQL keywords, so prefer `asc` over `ASC`

Reply:
Sorry, but I did google and uppercase should be right. 
Reference: https://www.w3schools.com/sql/sql_orderby.asp
https://www.geeksforgeeks.org/sql-order-by/
http://www.sql-tutorial.com/sql-order-by-sql-tutorial/

Reply:
SQL is case-insensitive, @drochetti's point was a style note. We prefer to use lowercase SQL keywords